docs/one-click-integrations/github-integration.md

```diff
@@ -46,6 +46,10 @@ The following is a detailed description of how the Pulse GitHub integration auto
 -   The set of changes in a deployment is the list of commits in the corresponding pull request. Pulse correctly tracks your changes even if you squash the commits when merging the pull request.
```

Comment:
I noticed that we already mentioned the squash merge above. :sweat_smile: What if we just expanded this bullet point to clarify a bit better?

```suggestion
-   The set of changes in a deployment is the list of commits in the corresponding pull request. Pulse correctly tracks your changes even if you [squash or rebase the commits when merging the pull request](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github), since Pulse processes all the original commits before any changes to the Git history.
```

Reply:
🤦